index.html

```diff
@@ -3,14 +3,15 @@
 <meta name="viewport" content="width=device-width, initial-scale=1">
 
 <head>
-    <title>Home - Ma page avec Bootstrap</title>
+    <title>Home</title>
     <link rel="stylesheet" href="css/bootstrap.min.css">
+    <link rel="stylesheet" href="css/style.css">
 </head>
 
 <body>
-    <nav class="navbar navbar-expand-lg bg-body-tertiary">
+    <nav class="navbar navbar-expand-lg bg-body-tertiary sticky-top">
         <div class="container-fluid">
-            <a class="navbar-brand" href="index.html">Ma page</a>
+            <a class="navbar-brand" href="index.html">A. Poirier Design Web</a>
```

Comment:
Try to put a padding or margin right. "Logo" is to near the border of the page. The same for all menu content.